controllers/chatController.js

```diff
@@ -121,7 +121,7 @@ exports.createChat = catchAsync(async (req, res, next) => {
   res.status(201).json({
     status: 'success',
     data: {
-      chat,
+      chat, //TODO: Remove this field
       lastMessage: message,
     },
```

Comment:
When?

Reply:
Not sure tbh, but that was supposed to be a temporary fix.

Reply:
Good, I thought that you forgot that line